io-engine/src/bin/io-engine-client/v1/snapshot_cli.rs

```diff
@@ -18,6 +18,8 @@ pub async fn handler(
 ) -> crate::Result<()> {
     match matches.subcommand() {
         ("create", Some(args)) => create(ctx, args).await,
```

Comment:
Need a suggestion here, if We need to keep separate `create` snapshot operation in CLI. as it will lead to confusion.
Currently create snapshot will be placed in `replica` and `nexus` section.

Reply:
Why not move creates to snapshot api then?
hmm alternatively starting to wonder if it's better to leave things as they are..